service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -36,6 +48,23 @@ default void bindServiceInfo(ServiceInfo serviceInfo) {
         binder.bind(JavadslServicesRouter.class);
     }
 
+    /**
+     * Binds Service interfaces with their implementations and registers them for publishing. This method must be
+     * invoked exactly one per Lagom service unless you are developing a consume-only service (see
+     * {@link ServiceGuiceSupport#bindServiceInfo} for consume-only service support).
+     * <p>
+     * Inspects all bindings and creates routes to serve every call described in the services bound.
+     * <p>
+     * The service bindings are used to build a {@link ServiceInfo} but only the <code>locatable</code>
+     * services will be part of the {@link ServiceInfo} metadata.
+     *
+     * @param serviceBindings an arbitrary list of {@link ServiceBinding}'s (use the convenience method
+     *                        {@link ServiceGuiceSupport#serviceBinding(Class, Class)} to build a
+     *                        {@link ServiceBinding}). Despite being a <code>varargs</code> argument, it is required to
+     *                        provide at least one {@link ServiceBinding} as argument, if you are building a Lagom
+     *                        service that acts only as a consumer use {@link ServiceGuiceSupport#bindServiceInfo}
```

Comment:
It doesn't seem right to have the alternate for consumers at the end, maybe at the beginning of the description state this is for services that publish and consume?